packages/common/src/content/_internal/computeProps.ts

```diff
@@ -41,6 +41,7 @@ export function computeProps(
   }
   // thumbnail url
   content.thumbnailUrl = getItemThumbnailUrl(model.item, requestOptions, token);
+  content.licenseInfo = model.item.licenseInfo || "";
```

Comment:
Is there a reason you are defaulting to `""`?

Reply:
`licenseInfo` can sometimes be null (I'm pretty sure)

Reply:
For sure, but since `null` and `""` are both falsy it shouldn't matter much. Unless we're getting errors because something is checking string length, we should be able to leave things as-is and not provide the fallback.